dist/angular-spring-data-rest.js

```diff
@@ -67,6 +67,8 @@ angular.module("spring-data-rest").provider("SpringDataRestAdapter", function ()
         },
 
         $get: ["$injector", function ($injector) {
+            // map contain self link as key and a list of linkNames which were already fetched
```

Comment:
I would write something like:

```
/**
* Link map which contains the 'self' link as key and the list of already fetched link names as value. This is used to 
* check that every link on each entity is only fetched once to avoid an infinite recursive loop.
*/
var linkMap = {}
```

Then it is more clear for that the map is used.